packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -20,11 +20,18 @@ export const Table = ({ data, height, onCellClick, width }: Props) => {
   const theme = useTheme();
   const [ref, headerRowMeasurements] = useMeasure();
   const tableStyles = getTableStyles(theme);
+  const defaultColumn = React.useMemo(
```

Comment:
nit: since this one never will change and alternative could be to declare it outside the render function.

Reply:
Will revert these changes, :man_facepalming: